src/js/components/Paragraph.js

```diff
@@ -31,5 +32,6 @@ Paragraph.propTypes = {
   align: PropTypes.oneOf(['start', 'center', 'end']),
   id: PropTypes.string,
   margin: PropTypes.oneOf(['none', 'small', 'medium', 'large']),
-  size: PropTypes.oneOf(['small', 'medium', 'large'])
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  width: PropTypes.oneOf(['small', 'medium', 'large'])
```

Comment:
@ericsoderberghp - I don't think that would be ideal.  I have some cases where it would be preferable to be able to have different text-sized paragraphs be of the same width, and automatically basing paragraph width off of text-size could mess up existing styling for anyone who's already using Grommet, since it wouldn't be backwards-compatible.